.github/workflows/weekly-summary.yml

```diff
@@ -0,0 +1,22 @@
+# Generate a weekly summary of test failures
+name: Weekly Summary
+on:
+  schedule:
+    # Run early Monday mornings, but long enough after daily test run for results
+    - cron: 0 4 * * 1
```

Comment:
I'd suggest `0 8 * * 1`, to ensure this runs late enough to catch also the "everything timed out" case.

The timeout for each job is 480 minutes = 8 hours, there are 3 workers to take 3 jobs from the matrix, things start at 23:00, so that would give a buffer of 1 hour for starting one of the runners later because somebody was in late or some such. Given when we usually start the work day, having results at 4 or 8 AM should be the same... @rvykydal?